### PR TITLE
Update the hello_world tutorial

### PR DESCRIPTION
**Pull request by rajarshimaitra**
Finishes up https://github.com/bitcoindevkit/bitcoindevkit.org/pull/93.

Couldn't get the link reference list thingy working for `SqliteDatabase` link so made an in place link.

### CODE REVIEW

File: docs/tutorials/hello-world.md [hunk 1]
```diff
@@ -180,17 +160,14 @@ println!(
...
 
-We briefly mentioned before that for our example we used the `MemoryDatabase`, but that it could also be swapped for a different one: this is one example of the *modularity* of BDK. By default, some database
-types are implemented in the library, namely (as of now) the `MemoryDatabase` which only keeps data in RAM and the [sled][sled] database that can store data on a filesystem. But since the `Database` trait is public,
-users of the library can also implement different database types more suitable for their use-case.
+We briefly mentioned before that for our example we used the `MemoryDatabase`, but that it could also be swapped for a different one: this is one example of the *modularity* of BDK. By default, some database types are already implemented in the library, namely the `MemoryDatabase` which only keeps data in RAM, the [sled][sled] database that can store data on a filesystem, and the [`SqliteDatabase`](https://docs.rs/bdk/latest/bdk/database/struct.SqliteDatabase.html) that can store data into a SQLite database. But since the `Database` trait is public, users of the library can also implement different database types more suitable for their use-case.
```
Comment (vladimirfomene) [inline docs/tutorials/hello-world.md]: @rajarshimaitra I was able to get the reference link working on my end using `[\`SqliteDatabase\`][SqliteDatabase]` in the text and `[SqliteDatabase]: https://docs.rs/bdk/latest/bdk/database/struct.SqliteDatabase.html` in the reference section at the end.

File: docs/tutorials/hello-world.md [hunk 2]
```diff
@@ -170,8 +147,11 @@ And then broadcast it:
 ```rust
 // ...
 
+use bdk::blockchain::Blockchain
```
Comment (vladimirfomene) [inline docs/tutorials/hello-world.md]: Nits: Missing semicolon

File: docs/tutorials/hello-world.md [hunk 3]
```diff
@@ -75,13 +68,11 @@ Building and running this code will print out:
...
 
-To fix this we can add one more component to our `Wallet`: a [`Blockchain`][Blockchain] backend. In particular, we are going to use the [`ElectrumBlockchain`][ElectrumBlockchain] which syncs with an `Electrum` server,
-since that's available out of the box in BDK and is pretty fast.
+To fix this we can add one more component in our code: a [`Blockchain`][Blockchain] backend. In particular, we are going to use the [`ElectrumBlockchain`][ElectrumBlockchain] which syncs with an `Electrum` server, and then we will use this blockchain to `sync` our wallet database with current state of the network since that's available out of the box in BDK and is pretty fast.
 
-We can change our `Wallet` construction to look something like this:
+We can update our code to look something like this:
 
 ```rust
 use bdk::{
```
Comment (vladimirfomene) [inline docs/tutorials/hello-world.md:86]: This import does not have `wallet::AddressIndex` like the import section above.

**Review by vladimirfomene — COMMENTED** [review]
I pulled the branch and built the wallet while following the tutorial. Tested ACK

File: docs/tutorials/hello-world.md (hunk 1 above)
Comment (rajarshimaitra) [reply docs/tutorials/hello-world.md]: Thanks, yes I updated the link and this time it seems to be working.. Can you check again?

File: docs/tutorials/hello-world.md (hunk 2 above)
Comment (rajarshimaitra) [reply docs/tutorials/hello-world.md]: Done..

File: docs/tutorials/hello-world.md (hunk 3 above)
Comment (rajarshimaitra) [reply docs/tutorials/hello-world.md:86]: Good catch.. Done..

**Comment by rajarshimaitra** [comment]
Thanks @vladimirfomene for the review.. I updated as suggest, rebased on master, and removed the old Hello World blog post.. Which was a previous duplicate of this tutorial.

**Comment by notmandatory** [comment]
Looks like this one is ready to go but needs a final rebase. Sorry for the long delay!

**Review by notmandatory — APPROVED** [review]
utACK 084e24e5a37beed913461c59693642747939f3cb

**Pull request merged**